added setSeconds (#2524)
Codecademy/docs: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md

---
Title: '.setSeconds()'

> Description: 'Set the seconds component of a date object to a specified value'

Reviewer: ```suggestion
Description: 'Sets the seconds component of a date object to a specified value.'
```

Subjects:
  - 'Web Development'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'paths/web-development'
  - 'introduction-to-javascript'
---

> The **`.setSeconds()`** method takes an integer value as its parameter, representing the new value for the seconds.

Reviewer: ```suggestion
The **`.setSeconds()`** method sets the seconds component of a date object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.
```

> The **`.setSeconds()`** method sets the seconds component of a date object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.

Reviewer 2: ```suggestion
The **`.setSeconds()`** method sets the seconds component of a `Date` object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.
```

## Syntax

```pseudo
myDate.setSeconds(secondsValue)
```

> The returned value from `.setSeconds()` modifies the original date object by setting the seconds to the specified value.

Reviewer: ```suggestion
The returned value from `.setSeconds()` modifies the original date object, `myDate`, by setting the seconds to `secondsValue`.
```

> The returned value from `.setSeconds()` modifies the original date object, `myDate`, by setting the seconds to `secondsValue`.

Reviewer 2: ```suggestion
The returned value from `.setSeconds()` modifies the original `Date` object, `myDate`, by setting the seconds to `secondsValue`.
```

## Example

In the example below, the `myDate` variable takes the provided Time and then the `.setSeconds()` method is applied to return a modified time.

> ```js

Reviewer: Can we use a static datetime value so the output will be consistent whenever we run this code?

> In the example below, the `myDate` variable takes the provided Time and then the `.setSeconds()` method is applied to return a modified time.

Reviewer 2: ```suggestion
In the example below, the `myDate` variable is declared and then the `.setSeconds()` method is applied to return a modified time.
```

> const myDate = new Date(August 29, 2022 23:15:30);

Reviewer 2: ```suggestion
const myDate = new Date('August 29, 2022 23:15:30');
```

myDate.setSeconds(42);

> console.log(myDate);

Reviewer 2: ```suggestion
console.log(myDate.toString());
```
